geoextent/helpfunctions.py

```diff
@@ -0,0 +1,100 @@
+
+import sys, os, platform, datetime, math, shapefile, fiona 
+
+import getopt
+from osgeo import ogr
+from osgeo import osr
+from pyproj import Proj, transform
+
+WGS84_EPSG_ID = 4326
+
+def printObject(object):
+    '''
+    Function purpose: output of metadata object \n
+    Input: object \n
+    Output: print("/n")
+    '''
+    print("\n")
+    for a,b in object.items():
+        print(str(a) + ": " + str(b))
+    print("\n")
+
+
+
+
+def exists(filename):
```

Comment:
Function does not add anything beyond the inner `os.path.isfile()` - stick to existing functions.